administrator/language/en-GB/en-GB.plg_quickicon_extensionupdate.ini

```
@@ -10,6 +10,6 @@ PLG_QUICKICON_EXTENSIONUPDATE_GROUP_DESC="The group of this plugin (this value i
 PLG_QUICKICON_EXTENSIONUPDATE_GROUP_LABEL="Group"
 PLG_QUICKICON_EXTENSIONUPDATE_UPDATEFOUND="Updates are available! %s"
 PLG_QUICKICON_EXTENSIONUPDATE_UPDATEFOUND_BUTTON="View Updates"
-PLG_QUICKICON_EXTENSIONUPDATE_UPDATEFOUND_MESSAGE="%s Extension Update(s) are available:"
+PLG_QUICKICON_EXTENSIONUPDATE_UPDATEFOUND_MESSAGE="%s Extension Update(s) are available"
```

This is now a sentence so should end with a full stop.

```suggestion
PLG_QUICKICON_JOOMLAUPDATE_UPDATEFOUND_MESSAGE="Joomla %s is available."
```